nais/labs-gcp.yaml

```diff
@@ -21,5 +21,8 @@ spec:
         initialDelay: 10
     accessPolicy:
         outbound:
+            rules:
+                - application: sykefravarsstatistikk-api
+                  namespace: arbeidsgiver
```

Comment:
Var det snakk om å bruke et eget namespace for vårt team? Ref https://trello.com/c/66W73Mz7
@erlend-axelsson, @thomasdufourd

Reply:
Da må vi oppdatere denne filen slik jeg har forstått: https://github.com/navikt/teams/blob/main/teams.yml

Ref: https://doc.nais.io/clusters/team-namespaces/

Reply:
Det virker som at vi har et team allerede, @navikt/teamia men den ligger ikke i den filen